pyspeckit/wrappers/cube_fit.py

```diff
@@ -74,7 +74,7 @@ def cube_fit(cubefilename, outfilename, errfilename=None, scale_keyword=None,
     # This speeds up the process enormously, but can easily mess up the fits if
     # there are bad pixels
     sp.momenteach(vheight=vheight, verbose=verbose)
-    sp.fiteach(errmap=errmap, multifit=True, verbose_level=verbose_level,
+    sp.fiteach(errmap=errmap, multifit=None, verbose_level=verbose_level,
```

Comment:
I get the feeling that I might have told you not to do this here; I think that's actually right... we can leave multifit in place here, sorry.  We should only remove it from the examples.